src/workerd/jsg/value.h

```diff
@@ -477,6 +478,35 @@ class StringWrapper {
     return wrap(context, creator, value.asPtr());
   }
 
+  template <StringLike T>
+  kj::Maybe<T> tryUnwrap(v8::Local<v8::Context> context, const v8::FastOneByteString& handle, T*) {
+    auto& js = Lock::from(context->GetIsolate());
+    kj::Array<char> buf;
+    if (simdutf::validate_ascii(handle.data, handle.length)) {
+      buf = kj::heapArray<char>(handle.length + 1);
+      buf.asPtr().copyFrom(kj::arrayPtr(handle.data, handle.length + 1));
+      buf[handle.length] = '\0';
```

Comment:
Is the original V8 string null-terminated? If it is, then line 488 isn't needed, and if it's not then isn't it an out-of-bounds read?

Reply:
v8::fastOnebytestring is not nul terminated